packages/cli-platform-android/src/commands/runAndroid/runOnAllDevices.ts

```diff
@@ -16,14 +16,17 @@ import tryLaunchAppOnDevice from './tryLaunchAppOnDevice';
 import tryLaunchEmulator from './tryLaunchEmulator';
 import tryInstallAppOnDevice from './tryInstallAppOnDevice';
 import {Flags} from '.';
+import {BuildFlags} from '../buildAndroid';
 
 export function getTaskNames(
   appName: string,
-  commands: Array<string>,
+  args: BuildFlags,
```

Comment:
I'm not a fan of leaking whole `args` to a helper function. Can we convert that into explicit arguments? Either as an object or a arg list, I'm fine with both